packages/adapter-node/src/index.js

```diff
@@ -42,19 +42,26 @@ if (socket_activation) {
 	});
 }
 
-function shutdown() {
+/** @param {'SIGINT' | 'SIGTERM' | 'IDLE'} reason */
+function graceful_shutdown(reason) {
 	if (shutdown_timeout_id) return;
 
 	// @ts-expect-error this was added in 18.2.0 but is not reflected in the types
 	server.server.closeIdleConnections();
```

Comment:
do we need this line? I think `.close` is going to close the idle connections anyway on the very next line, so this doesn't do anything
```suggestion
```

Reply:
See my comment here https://github.com/sveltejs/kit/pull/12153#discussion_r1638836770